composer.json

```diff
@@ -71,7 +71,7 @@
     "require": {
         "php":                          ">=5.3.3",
         "symfony/framework-bundle":     "~2.3",
-        "symfony/security-bundle":      "~2.1",
+        "symfony/security-bundle":      "~2.6",
```

Comment:
There is no option as this patch is intended to do so. Using ```security.context``` triggers errors forcibly. With this patch I want to prepare bundle for further migrations. As noted in PR comment we can either create a transition branch for this patch or create a branch for previous version (as was made before)